homeassistant/components/jellyfin/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""The Jellyfin integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.jellyfin.config_flow import (
+    CannotConnect,
+    InvalidAuth,
+    validate_input,
+)
+from homeassistant.components.jellyfin.const import DATA_CLIENT, DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Jellyfin from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    try:
+        client = await validate_input(hass, entry.data)
+    except CannotConnect:
+        _LOGGER.error("Cannot connect to Jellyfin server")
+        return False
```

Comment:
`homeassistant.exceptions.ConfigEntryNotReady`

```suggestion
    except CannotConnect as ex:
        raise ConfigEntryNotReady from ex
```

Reply:
```suggestion
    except CannotConnect as ex:
            raise ConfigEntryNotReady("Cannot connect to Jellyfin server") from ex
```

Best would be to include the message, if CannotConnect doesn't have a clear exception message set. In a future version this will be shown in the front-end.